src/wallet/load.cpp

```diff
@@ -22,6 +23,17 @@
 #include <system_error>
 
 namespace wallet {
+
+bool HandleWalletLoadError(interfaces::Chain& chain, const std::string& wallet_file, const bilingual_str& error_string)
+{
+    if (!chain.initQuestion(error_string + Untranslated("\n\n") + _("Continue without wallet?"), error_string, _("Error"), CClientUIInterface::MSG_ERROR | CClientUIInterface::MODAL | CClientUIInterface::BTN_OK | CClientUIInterface::BTN_ABORT)) {
```

Comment:
Maybe better question message would be "Continue without this wallet?", so that is clear that only wallet having error is not being loaded, not all other wallets or wallet functionality disabled altogether?

Reply:
Sounds good, done